### PR TITLE
Added CD and improve CI

### PR DESCRIPTION
**Pull request by UlisesGascon**
### Main Changes
- Added support for NVM (db9de93)
- Added npm command `lint:fix` (a953851)
- Fix typo in `contributors` metadata field in the `package.json` (2c06636)
- Automated NPM publications with GA pipeline (6930981)
- Updated CI pipeline:
  - Changed triggers policy
  - Added support for `Node@20.x` and `Node@21.x` (9260237)

### Changelog

- 9260237 feat: add support for Node@20.x and Node@21.x in CI pipeline by @UlisesGascon
- 9722345 feat: add policies to run CI by @UlisesGascon
- 6930981 feat: add NPM publication pipeline by @UlisesGascon
- e37536c fix: update CI pipeline to use the default branch by @UlisesGascon
- 2c06636 fix: typo in pkg metadata by @UlisesGascon
- a953851 chore: add npm command lint:fix by @UlisesGascon
- 48bf225 chore: lint files by @UlisesGascon
- db9de93 feat: add support for NVM by @UlisesGascon

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+    branches: [ master ]
 jobs:
   build:
     name: Build, lint, and test on Node ${{ matrix.node }} and ${{ matrix.os }}
 
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node: ["18.x"]
+        node: [18.x, 20.x, 21.x]
```
Comment (telekosmos) [inline .github/workflows/main.yml:14]: 👍

Comment (UlisesGascon) [reply .github/workflows/main.yml:14]: yeah xD

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@onebeyond/notion-exporter",
   "author": "Yann Bolliger",
-  "contributors: ["One Beyond"],
+  "contributors": ["One Beyond"],
```
Comment (telekosmos) [inline package.json:4]: I don't know how this passed previously 😨

Comment (UlisesGascon) [reply package.json:4]: I accidentally introduced that on #2 as I made a code suggestion on the web and I didn't check that locally once I resolved the suggestion before merging the PR.

**Pull request merged**